spec/notification.js

```diff
@@ -188,6 +188,20 @@ describe('notification api', () => {
         notifyClient.prepareUpload(file);
       }).to.throw("File is larger than 2MB.")
     });
+
+    it('should allow send a file email confirmation to be set', () => {
+      let file = Buffer.alloc(2*1024*1024)
+      expect(
+        notifyClient.prepareUpload(file, {confirmEmailBeforeDownload: true})
+      ).contains({confirm_email_before_download: true})
+    });
+
+    it('should allow custom retention periods to be set', () => {
+      let file = Buffer.alloc(2*1024*1024)
+      expect(
+        notifyClient.prepareUpload(file, {retentionPeriod: "52 weeks"})
+      ).contains({retention_period: '52 weeks'})
+    });
```

Comment:
There are tests elsewhere already that cover the current/'old' `isCsv` case, eg notification.js:120-132.

We can add explicit ones here as well though if you want. I'm not sure we want to ever not send `is_csv` - at least, the current client always sends it so it would maybe be weird for us to stop sending it now?

Reply:
yeah, lets add explicit ones here. it's a new way of passing in the isCsv flag that we now support, we should make sure it's appropriately tested.

lets discuss whether is_csv should be allowed to exist or not in a diff comment.